docs/build-a-custom-amm/build-an-amm/create-custom-amm-with-novel-invariant.md

```diff
@@ -321,7 +321,7 @@ Balancer protocol supports two types of swaps:
 - `EXACT_IN` - The user defines the exact amount of `tokenIn` they want to spend.
 - `EXACT_OUT` - The user defines the exact amount of `tokenOut` they want to receive.
 
-The `minAmountOut` or `maxAmountIn` are enforced by the [vault](https://github.com/balancer/balancer-v3-monorepo/blob/main/pkg/vault/contracts/Vault.sol#L405) .
+The `minAmountOut` or `maxAmountIn` are enforced by the [vault](https://github.com/balancer/balancer-v3-monorepo/blob/main/pkg/vault/contracts/Vault.sol#L391-L393) .
```

Comment:
@EndymionJkb You know the vault in your sleep. I would need to read through it to find that check again. Which lines are you refering to here?

Reply:
Missed that link. I'll find and update it.